locations/spiders/bricorama_fr.py

```
@@ -0,0 +1,27 @@
+
+from locations.categories import apply_category, Categories
+from locations.items import Feature
+from locations.structured_data_spider import StructuredDataSpider
+from scrapy.linkextractors import LinkExtractor
+
+
+class BricoramaFRSpider(CrawlSpider, StructuredDataSpider):
+    name = "bricorama_fr"
+    item_attributes = {"brand": "Bricorama", "brand_wikidata": "Q2925146"}
+    start_urls = ["https://www.bricorama.fr/magasins?device=mobile"]
+    rules = [Rule(LinkExtractor(r"/magasin/[^/]+/(\d+)$"), "parse")]
```

Ruff (0.16.1)

[warning] Mutable default value for class attribute

(RUF012)